File: bachelor-thesis-odms/Couchbase/JavaODMs/odmstest/src/main/java/cz/cuni/mff/couchbase_java/springdata_e/models/RegionEOnlyName.java
package cz.cuni.mff.couchbase_java.springdata_e.models;

public class RegionEOnlyName {
    private int r_regionkey;
    private String r_name;

    public int get_r_regionkey() {
        return r_regionkey;
    }

    public RegionEOnlyName() {}

    public RegionEOnlyName(int r_regionkey, String r_name) {
        this.r_regionkey = r_regionkey;
        this.r_name = r_name;
    }
}

File: evaluation/common-csharp/Models/TPC_H/E/RegionEOnlyName.cs
namespace CommonCSharp.Models.TPC_H
{
    public class RegionEOnlyName
    {
        public int r_regionkey { get; set; }
        public string r_name { get; set; }

        public RegionEOnlyName() { }

        public RegionEOnlyName(int r_regionkey, string r_name)
        {
            this.r_regionkey = r_regionkey;
            this.r_name = r_name;
        }
    }
}
